taco-tuesday-rest-api/src/main/java/com/muskopf/tacotuesday/domain/IndividualOrder.java

```diff
@@ -1,25 +1,32 @@
 package com.muskopf.tacotuesday.domain;
 
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+
 import javax.persistence.*;
 
 @Entity
+@JsonIdentityInfo(
+        generator = ObjectIdGenerators.PropertyGenerator.class,
+        property = "id"
+)
 public class IndividualOrder extends Order {
-    @ManyToOne(cascade = CascadeType.ALL)//fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "fk_full_order_id", referencedColumnName = "id")
+    @JsonBackReference
     private FullOrder fullOrder;
 
-    @ManyToOne(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    @ManyToOne(fetch = FetchType.EAGER, cascade = CascadeType.MERGE)
```

Comment:
`CascadeType.MERGE` does more of what we're looking for [here](https://www.concretepage.com/hibernate/example-cascadetype-merge-hibernate).